internal/jsonwall/log.go

```diff
@@ -0,0 +1,53 @@
+package jsonwall
+
+import (
+	"fmt"
+	"sync"
+)
+
+// Avoid importing the log type information unnecessarily.  There's a small cost
```

Comment:
Multiple spaces between sentences - some more occurrences in this file.

Reply:
Sorry, personal style, but I may need to reconsider it. Is gofmt complaining about this?

With that said, I'd prefer to not touch this file here. This is a copy & paste from other log.go files in other packages, and the duplication is a way to give the independent packages a way to hook up into central logging without making every package depend on the central logging (easy to reuse it across projects).

Reply:
gofmt does not seem to mind :)